engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -164,4 +160,9 @@ public void onMouseX(MouseAxisEvent event, EntityRef entity) {
         }
     }
 
+    private void timeTravel(EntityRef entity, KeyEvent event, float timeInDays) {
```

Comment:
Protip: even if this is a private method it is a good place to add Javadoc to make it clear what the method does :-)